workspaces/orchestrator/plugins/orchestrator-form-widgets/README.md

```diff
@@ -0,0 +1,70 @@
+# orchestrator-form-widgets
+
+This Backstage frontend plugin provides default, but optional, set of RJSF form widgets for the Orchestrator workflow execution page.
+
+## Deployment
+
+### Static (for development)
+
+In the packages/app/src/App.tsx:
+
+```
+import { orchestratorFormWidgetsPlugin } from '@red-hat-developer-hub/backstage-plugin-orchestrator-form-widgets';
+...
+const app = createApp({
+    ...
+    plugins: [orchestratorFormWidgetsPlugin],
+})
+```
+
+### Dynamic (for RHDH production)
+
+For RHDH production deployments, it is expected that the plugin is exported as a dynamic plugin using janus CLI a loaded among the other dynamic frontend plugins.
+No explicit configuration is needed.
+
+## Content
+
+The plugin provides implementation of `OrchestratorFormApi` (for `orchestratorFormApiRef`) to extend the Workflow execution form for custom provided ui:widgets.
+
+## Context
+
+## SchemaUpdater widget
+
+A headless widget used for fetching snippets of JSON schema and dynamically updating the RJSF form JSON schema on the fly.
+
+Example of use in workflow's input data schema:
+
+```
```

Comment:
nit: to improve readability, you can add "```json"

Reply:
done